media/js/firefox/firstrun/firstrun_quantum.js

```diff
@@ -5,6 +5,11 @@
 (function (Mozilla) {
     'use strict';
 
+    // Add class to indicate Windows 8 and later
+    if ((window.site.getPlatform() === 'windows') && (parseFloat(window.site.getPlatformVersion()) >= 6.2)) {
+        document.documentElement.className += ' win8up';
```

Comment:
As this is Firefox only, you can use `document.documentElement.classList.add('win8up');`